tailwind.config.js

```diff
@@ -96,10 +112,22 @@ module.exports = {
                 '5.5xl': '3.5rem',
                 '6.5xl': ['4rem', '1.10'],
                 '7.5xl': '5rem',
+                  '13'  : '13px',
+                  '15'  : '15px',
+                  '32'  : '32px',
             },
 
 
-            
+            borderWidth:{
+              3:'3px',
+              5:'5px',
+              6:'6px',
+            },
+            ringWidth: {
+              3:'3px',
+              5:'5px',
+              6:'6px',
+            },
```

Comment:
No `px` values only use `rem`.

Reply:
Ring width and Border width should be used in fixed format TailwindCSS also used border in px instead of rem. We added these values cause TailwindCSS does not have these defaults.
![image](https://user-images.githubusercontent.com/5028960/207413408-37ea95ad-e11a-4104-86f3-8f54d4cb7c38.png)

Reply:
that's fine you can covert using this https://nekocalc.com/px-to-rem-converter